opcua/status_code.py

```diff
@@ -663,4 +663,4 @@ def get_name_and_doc(val):
     elif val == 0x80B70000:
         return 'BadMaxConnectionsReached', 'The operation could not be finished because all available connections are in use.'
     else:
-        return 'UknownUaError', 'Unknown StatusCode value: {}'.format(val)
+        return 'UnknownUaError', 'Unknown StatusCode value: {}'.format(val)
```

Comment:
as the first line states this file is autogenerated :-), you need to fix generate_statuscodes.py under schemas